File: VB/MainForm.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace T223550
	Partial Public Class MainForm
		Inherits XtraForm

		Public Sub New()
			InitializeComponent()
			mvvmContext1.SetBinding(luMaster.Properties, Function(p) p.DataSource, "Customers")
			mvvmContext1.SetBinding(luMaster, Function(p) p.EditValue, "SelectedCustomer")

			mvvmContext1.SetBinding(luChild.Properties, Function(p) p.DataSource, "Orders")
			mvvmContext1.SetBinding(luChild, Function(p) p.EditValue, "SelectedOrder")
		End Sub
	End Class
End Namespace

File: CS/T223550/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace T223550 {
    public partial class MainForm : XtraForm {
        public MainForm() {
            InitializeComponent();
            mvvmContext1.SetBinding(luMaster.Properties, p => p.DataSource, "Customers");
            mvvmContext1.SetBinding(luMaster, p => p.EditValue, "SelectedCustomer");

            mvvmContext1.SetBinding(luChild.Properties, p => p.DataSource, "Orders");
            mvvmContext1.SetBinding(luChild, p => p.EditValue, "SelectedOrder");
        }
    }
}
